app/views/layouts/application.haml

```diff
@@ -0,0 +1,10 @@
+!!!
+%html
+  %head
+    %meta{content: "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace